docs/deploying.md

```diff
@@ -48,7 +48,7 @@ unleash-edge edge --cors-origin "https://example.com" --cors-origin "https://oth
 Edge provides a range of powerful ways in which you can run it. For a standard production configuration we recommend the
 following:
 
-- [Run in Edge mode](#edge): Edge mode connects to your upstream Unleash and syncs feature flags and tokens. This should
```

Comment:
This anchor doesn't exist. Not sure what the original intent was

Reply:
I'm pretty sure this doc was extracted from the README, where the anchor for edge mode was available. We can remove it now